Makefile

```diff
@@ -107,6 +107,7 @@ PROTOC_VERSION := 3.6.1
 PROTOC_PGV_VERSION := v0.3.0-java.0.20200311152155-ab56c3dd1cf9
 GOLANG_PROTOBUF_VERSION := v1.3.2
 GOLANGCI_LINT_VERSION := v1.26.0
+GINKGO_VERSION := v1.12.0
```

Comment:
Can we lock the version to v1.12.2?

Reply:
We can't because it upgrades `github.com/golang/protobuf` which breaks the project. I'll let it for a person who will be upgrading dependencies just like we said we will be doing monthly.